app/views/rentalsandrentaroom/adjustments/BusinessPremisesRenovationAllowanceBalancingChargeView.scala.html

```diff
@@ -0,0 +1,74 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.govuk._
+@import controllers.rentalsandrentaroom.adjustments.routes.BusinessPremisesRenovationBalancingChargeController
+@import viewmodels.LabelSize
+@import viewmodels.InputWidth._
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+)
+
+@(form: Form[_], taxYear: Int, individualOrAgent: String, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("businessPremisesRenovationBalancingCharge.title"))) {
+
+<h1 class="govuk-heading-l">@messages("businessPremisesRenovationBalancingCharge.heading")</h1>
+
+    @formHelper(action = BusinessPremisesRenovationBalancingChargeController.onSubmit(taxYear, mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+
+        <div>
+            <p class="govuk-body">@messages("businessPremisesRenovationBalancingCharge.details.content.line1")</p>
+            <p class="govuk-body">@messages("businessPremisesRenovationBalancingCharge.details.content.line2")</p>
+        </div>
+
+        @govukRadios(
+                radios.RadiosViewModel.yesNoWithConditionalHtml(
+                    field = form("businessPremisesRenovationAllowanceBalancingChargeYesOrNo"),
+                    legend = LegendViewModel(messages(s"businessPremisesRenovationAllowanceBalancingChargeAmount.radio.legend.$individualOrAgent")).withCssClass("govuk-fieldset__legend govuk-fieldset__legend--m"),
+                    items = radios.RadiosViewModel.yesNoItemsWithConditionalHtml(
+                        field = form("businessPremisesRenovationAllowanceBalancingCharge"),
+                        conditionalYesHtml = Some(
+                            govukInput(
+                                InputViewModel(
+                                    field = form("businessPremisesRenovationAllowanceBalancingChargeAmount"),
+                                    label = LabelViewModel(messages("businessPremisesRenovationAllowanceBalancingChargeAmount.radio.question.label"))
+                                )
+                                .asNumeric()
```

Comment:
.withHint missing?

Reply:
Not in the screenshot nor in the prototype so 🤷 - https://income-tax-submission.herokuapp.com/v06/6-2-10/property/property-rentals/uk/adjustments/opr-business-premises-renovation-allowance-balancing-charge